### PR TITLE
Added 5 scrum infotips or concepts and modified SDD infotip as per…

### PR DESCRIPTION
**Pull request by Preeticp**
…the one on the UX.
Please do not review. This has been sent to the devs for tech review, hope to get comments by tomorrow eod, will add final touches then.
Note: This PR adds 5 scrum infotips. However, I am yet to receive confirmation on the need for Issues and Impediment work item type hence have not added uuids for them and have commented them out.

### CODE REVIEW

**Comment by Preeticp** [comment]
We are good to go on this. Note we do not have Issues in the UX yet, but it will be introduced in the coming sprints. Similarly with Impediments, we do not see this on the UX, so keeping both of them commented out.  We might need it for glossary, even though we might not need it for infotips now.

File: docs/topics/modules/glossary.adoc [hunk 1]
```diff
@@ -79,6 +94,10 @@ papercut:: A papercut is a minor issue, often too small to be individually prior
 pipeline:: Pipelines are a continuous delivery system that, at each step, tests and deploys the code to provide feedback to the user. Examples of such steps are unit testing, performance, integration, and deployment. Each step of the pipeline implements different levels of testing and deployment tasks, provides results, and then passes the code on to the next step.
 // endterm
 
+// term: 58cb3a32-65d9-4087-afbd-4d91035c8e9f, en_EN
+product backlog item:: A small unit of work that can be completed by a team within an iteration and that can be broken down into tasks. It is commonly described in the format of an user story, with clearly specified requirements, an owner, an estimated size and ordered as per the order of execution.
```
Comment (rkratky) [inline docs/topics/modules/glossary.adoc]: - Why not just 'backlog item'?
-> a user story
-> size, and

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-310-2 is available for testing at http://fabric8-online-docs-app-pr-310-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-310-3 is available for testing at http://fabric8-online-docs-app-pr-310-fabric8-online-docs.badger.fabric8.io

**Pull request merged**